cypress/e2e/opening-hours-editor.cy.ts

```diff
@@ -8,10 +8,12 @@ enum OpeningHourCategories {
   PhoneTime = "Telefontid",
 }
 
+type TimeString = `${number}:${number}`;
+
 type OpeningHourFormType = {
   openingHourCategory: OpeningHourCategories;
-  start: `${number}:${number}`;
-  end: `${number}:${number}`;
+  start: TimeString;
+  end: TimeString;
```

Comment:
:+1: Nice to add a proper type for this.

You could consider creation a new `TimeDuration` type which is a pair of time strings with a start and an end. The values seems to be passed around in pairs anyway.